pom.xml

```diff
@@ -436,6 +454,13 @@
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>${version.surefire.plugin}</version>
+                    <dependencies>
+                        <dependency>
+                            <groupId>org.ow2.asm</groupId>
+                            <artifactId>asm</artifactId>
+                            <version>6.2</version>
```

Comment:
Can that version be extracted into a property, as it's used multiple times. And out of curiousity, why is it needed at all?

Reply:
@gunnarmorling i think it’s safe to remove. I had seen some places online where that was added. I had hoped it might help with the debezium embedded build issues. I’m gonna pull it out i think. 

- [ ] verify if necessary. If not remove, if it is extract version.